src/index.js

```diff
@@ -2907,6 +2907,19 @@ const createScatterplot = (
     pubSub.publish('destroy');
     pubSub.clear();
   };
+  const getPointsInView = () => {
+    const pointsInBBox = searchIndex.range(
+      bottomLeftNdc[0],
+      bottomLeftNdc[1],
+      topRightNdc[0],
+      topRightNdc[1]
+    );
+    const pts = [];
+    pointsInBBox.forEach((idx) => {
+      pts.push(idx);
+    });
```

Comment:
Ok loop now makes more sense, please check. Idea from the start was to return absolute position of the elements too so that textual DOM elements could be positioned absolutely in the current view over the scatterplot dots.